systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -118,20 +119,33 @@ void testMirrorMaker2(ExtensionContext extensionContext) {
         String topicSourceNameMirrored = kafkaClusterSourceName + "." + sourceTopicName;
 
         // Deploy source kafka
-        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(kafkaClusterSourceName, 1, 1).build());
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(kafkaClusterSourceName, 1, 1)
+            .editMetadata()
+                .withNamespace(namespaceName)
+            .endMetadata().build());
```

Comment:
As I mentioned on offline discussion, we should add the 
```
.editMetadata()
     .withNamespace(namespaceName)
.endMetadata()
```
to resource manager, where it will be added by the `extensionContext`, if it is possible.

Reply:
Yes, I have implemented the `ExtensionContextStorage.class` which will cover my next PR.